sarif-parser/tests/cli_test.py

```diff
@@ -0,0 +1,143 @@
+import json
+from pathlib import Path
+
+import pytest
+from sarif_parser.cli import cli
+
+sarif_json = """
+{
+  "runs": [
+    {
+      "results": [
+        {
+          "message": {
+            "text": "Container has no configured security context"
+          },
+          "level": "error",
+          "locations": [
+            {
+              "physicalLocation": {
+                "region": {
+                  "snippet": {}
+                },
+                "artifactLocation": {
+                  "uri": "/workspace/git/myproject/asd.yaml"
+                },
+                "contextRegion": {
+                  "snippet": {},
+                  "startLine": 1
+                }
+              }
+            }
+          ],
+          "properties": {
+            "issue_confidence": "HIGH",
+            "issue_severity": "HIGH"
+          },
+          "ruleId": "container-security-context-user-group-id"
+        },
+        {
+          "message": {
+            "text": "CPU limit is not set"
+          },
+          "level": "error",
+          "locations": [
+            {
+              "physicalLocation": {
+                "region": {
+                  "snippet": {}
+                },
+                "artifactLocation": {
+                  "uri": "/workspace/git/myproject/asd.yaml"
+                },
+                "contextRegion": {
+                  "snippet": {},
+                  "startLine": 1
+                }
+              }
+            }
+          ],
+          "properties": {
+            "issue_confidence": "HIGH",
+            "issue_severity": "HIGH"
+          },
+          "ruleId": "container-resources"
+        }
+      ]
+    }
+  ],
+  "version": "2.1.0",
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json"
+}
+"""
+expected_issues = [
+    {
+        "issue_code": "KUBESCORE-W1001",
+        "issue_text": "Container has no configured security context",
+        "location": {
+            "path": "workspace/git/myproject/asd.yaml",
+            "position": {
+                "begin": {"line": 1, "column": 1},
+                "end": {"line": 1, "column": 1},
+            },
+        },
+    },
+    {
+        "issue_code": "container-resources",
```

Comment:
Is this made up, or are we missing this in the mapping?

Reply:
`sarif-parser` tests don't deal with the mapping. Tests regarding that will be in the top level `tests` folder.